extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -30,7 +30,9 @@
     public Map<String, OidcTenantConfig> namedTenants;
 
     /**
-     * Default TokenIntrospection and UserInfo Cache configuration which is used for all the tenants if it is enabled.
+     * Default TokenIntrospection and UserInfo Cache configuration which is used for all the tenants if it is enabled
+     * with the build-time 'quarkus.oidc.default-token-cache-enabled' property ('true' by default) and also activated,
+     * see its `max-size` property.
```

Comment:
Sorry I think this is too schizophreniac for me: the cache is enabled by default, but also disabled because its max-size is 0. I have no idea how you can justify this.

Reply:
Hi @FroMage Sure lets continue, I'll be happy to adjust, if we agree on what has to be adjusted :-)
So the build time enable property is only about making this default cache available, it is like we have `enabled` property for different extensions, incl. OIDC. So it makes this bean available around, no more. 
We can't have it starting accumulating the data OOB - it would be a security risk - the users have to decide themselves if they'd like to cache - i.e, avoid re-verifying the token or refreshing the user info.
So this bean is around, and now if they users want to activate it then they'd to configure the size. I've checked the docs, everywhere there is a distinction between enabling and activating it.

How would you expect it to be enabled/activated in the configuration ? Please keep it in mind - it can't get the data by default, the data  size have to be constrained.

Reply:
@FroMage,

```
@ConfigRoot
public class OidcBuildTimeConfig {
    ...
    /**
     * Enable the registration of the Default TokenIntrospection and UserInfo Cache implementation bean.
     * Note it only allows to use the default implementation, one needs to configure it in order to activate it,
     * please see {@link OidcConfig#tokenCache}.
     */
    @ConfigItem(defaultValue = "true")
    public boolean defaultTokenCacheEnabled;
}
```

Does it clarify it ? It s really the same way with `quarkus-oidc` and other extensions where we can enable it (default is enabled) but unless you configure them they won't do any useful work...